CondTools/Geometry/test/writehelpers/createExtended2021DD4hepPayloads.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+
+if [ $# -ne 1 ]
+then
+  echo Error: createExtended2021Payloads.sh requires exactly one argument which is the tag
+  exit 1
+fi
+mytag=$1
+echo ${mytag}
+
+# Set the tag in all the scripts and the metadata text files
+sed -i {s/TagXX/${mytag}/g} *.py
+compgen -G "*.txt" > /dev/null && sed -i {s/TagXX/${mytag}/g} *.txt
+sed -i {s/TagXX/${mytag}/g} splitExtended2021Database.sh
+
+# First read in the little XML files and create the
+# large XML file for the Phase1_R30F12_HCal Ideal scenario.
+# Input cff                                             Output file
+# GeometryExtended2021_cff                       geSingleBigFile.xml
+cmsRun geometryExtended2021DD4hep_xmlwriter.py
+
+# Now convert the content of the large XML file into
+# a "blob" and write it to the database.
+# Also reads in the little XML files again and fills
+# the DDCompactView. From the DDCompactView the
+# reco parts of the database are also filled.
+cmsRun geometryExtended2021DD4hep_writer.py
+
+# Now put the other scenarios into the database.
```

Comment:
@cvuosalo - please, remove commented out code.

Reply:
As mentioned in the PR description, a forthcoming PR will enable these lines to produce the reduced material scenarios. Right now the lines serve as reminders of what will be implemented soon.